### PR TITLE
Puts a 24h expiration on the landing page results for YouTube + Poly

### PR DESCRIPTION
**Pull request by gfodor**

### CODE REVIEW

File: lib/ret_web/controllers/api/v1/media_search_controller.ex [hunk 1]
```diff
@@ -29,12 +29,19 @@ defmodule RetWeb.Api.V1.MediaSearchController do
...
   end
 
-  def index(conn, %{"source" => "assets", "user" => user} = params) do 
+  def index(conn, %{"source" => "assets", "user" => user} = params) do
     account = conn |> Guardian.Plug.current_resource()
 
     if account.account_id == String.to_integer(user) do
       {:commit, results} =
-        %Ret.MediaSearchQuery{source: "assets", user: account.account_id, type: params["type"], q: params["q"], cursor: params["cursor"] || "1"} |> Ret.MediaSearch.search()
+        %Ret.MediaSearchQuery{
```
Comment (gfodor) [inline lib/ret_web/controllers/api/v1/media_search_controller.ex]: formatter

File: lib/ret_web/controllers/api/v1/media_search_controller.ex [hunk 2]
```diff
@@ -64,6 +71,13 @@ defmodule RetWeb.Api.V1.MediaSearchController do
     end
   end
 
+  # For Google services, increase cache duration for landing pages by using long-lived cache, due to quotas.
+  defp cache_for_query(%Ret.MediaSearchQuery{source: source, q: nil})
+       when source == "youtube_videos" or source === "poly",
```
[the remark this answers is not in the document]
Comment (gfodor) [reply lib/ret_web/controllers/api/v1/media_search_controller.ex]: haha nice catch, I don't know what that will do!

[the remark this answers is not in the document]
Comment (brianpeiris) [reply lib/ret_web/controllers/api/v1/media_search_controller.ex]: it works! strict equality

[the remark this answers is not in the document]
Comment (gfodor) [reply lib/ret_web/controllers/api/v1/media_search_controller.ex]: `The difference between == and === is that the latter is more strict when comparing integers and floats:` would have been fine, but not intentional!

**Pull request merged**